src/licensedcode/data/licenses/ccl.yml

```diff
@@ -0,0 +1,5 @@
+key: cockroach
```

Comment:
You need to also rename the file for cockroach.yml and the text to cockroach.LICENSE